### ISSUE TITLE
Surprise is null error

### ISSUE THREAD

**Issue by andris155** [SickSkillz/SLB-Tracker#10]
**Describe the bug**
Surprise is null error

**To Reproduce**
Break Lucky Block (?)

**Expected behavior**
No write error

**Screenshots**

**Server (please complete the following information):**
 - OS: Ubuntu 18.04
 - Java version: 11
 - Minecraft version: 1.13.2

**Plugin (please complete the following information):**
 - Version: v2.13.5
 - Changed config: yes
 - Added or removed custom surprises: yes

**Additional notes**
```
[03:05:51] [Server thread/ERROR]: Could not pass event BlockBreakEvent to SuperLuckyBlock-Pro v2.13.5
org.bukkit.event.EventException: null
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.13.2.jar:git-Paper-486]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-486]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-486]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.breakBlock(PlayerInteractManager.java:299) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:226) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1291) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:40) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:10) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-486]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-486]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException: Surprise is null
	at org.sickskillz.superluckyblock.c.a(InGameUtils.java:369) ~[?:?]
	at org.sickskillz.superluckyblock.za.a(BlockBreakListener.java:112) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor477.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.13.2.jar:git-Paper-486]
	... 17 more

```

**Comment by SickSkillz** [comment]
Please install version 2.14.2 and see if you get the error again. If you get it, please send it to me again. I added more information to it.

https://www.spigotmc.org/resources/superluckyblock-1-8-1-13-the-luckyblock-plugin-that-gives-you-full-control.48837/update?update=258106

**Comment by andris155** [comment]
```
[07:46:42] [Server thread/ERROR]: Could not pass event BlockBreakEvent to SuperLuckyBlock-Pro v2.14.2
org.bukkit.event.EventException: null
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.13.2.jar:git-Paper-486]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-486]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-486]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.breakBlock(PlayerInteractManager.java:299) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:226) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1291) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:40) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:10) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-486]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-486]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-486]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException: Surprise doesn't exist: Winter-RandomFireworks
	at org.sickskillz.superluckyblock.n.g(InGameUtils.java:369) ~[?:?]
	at org.sickskillz.superluckyblock.ha.g(BlockBreakListener.java:120) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor477.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.13.2.jar:git-Paper-486]
	... 17 more
```

**Comment by SickSkillz** [comment]
Fixed in version 2.15.0: https://www.spigotmc.org/resources/superluckyblock-1-8-1-13-the-luckyblock-plugin-that-gives-you-full-control.48837/update?update=258437
